pkgs/by-name/ca/canvasdraw/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+
+  lsb-release,
+  which,
+  pkg-config,
+  gtk3,
+  fontconfig,
+  ftgl,
+  libGLU,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "canvasdraw";
+  version = "5.14";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/canvasdraw/${version}/Docs%20and%20Sources/cd-${version}_Sources.tar.gz";
+    sha256 = "sha256-xE7cBBExyaakKTPIeaZE1r2CjWCyikuFWpzeYV/Fr08=";
+  };
+
+  buildInputs = [
+    lsb-release
+    which
+    pkg-config
+    gtk3
+    fontconfig
+    ftgl
+    libGLU
+  ];
+
+  sourceRoot = "cd/src";
+
+  makeFlags = [
+    "USE_PKGCONFIG=Yes"
+    "USE_GTK3=Yes"
+    "cd"
+    "cdgl"
+    "cdcontextplus"
```

Comment:
What about other targets:
```makefile
do_all:        cd cdpdf cdgl cdim cdcontextplus cdlua5 cdluapdf5 cdluagl5 cdluacontextplus5 cdluaim5
```

Reply:
> What about other targets:
> 
> ```makefile
> do_all:        cd cdpdf cdgl cdim cdcontextplus cdlua5 cdluapdf5 cdluagl5 cdluacontextplus5 cdluaim5
> ```

I figure that they weren't added originally because they weren't needed but I am not sure. Should I add those other targets?